RecoTracker/LST/interface/LSTOutput.h

```diff
@@ -16,16 +18,14 @@ class LSTOutput {
         seedIdx_(std::move(seedIdx)),
         trackCandidateType_(std::move(trackCandidateType)) {}
 
-  enum LSTTCType { T5 = 4, pT3 = 5, pT5 = 7, pLS = 8 };
```

Comment:
if you wanted to have fewer updates, just `using LSTTCType = lst::LSTObjType;` should be enough here